dashboards/support.jsonnet

```diff
@@ -8,210 +8,281 @@ local row = grafonnet.panel.row;
 
 local common = import './common.libsonnet';
 
-// NFS Stats
-local userNodesNFSOps =
+
+// NFS usage diagnostics
+// ---------------------
+local nfsReqPerNode =
   common.tsOptions
-  + ts.new('User Nodes NFS Ops')
+  + ts.new('NFS requests, per node')
   + ts.standardOptions.withDecimals(0)
   + ts.queryOptions.withTargets([
     prometheus.new(
       '$PROMETHEUS_DS',
       |||
-        sum(rate(node_nfs_requests_total[5m])) by (node) > 0
+        sum(irate(node_nfs_requests_total[5m])) by (node) > 0
       |||
     )
     + prometheus.withLegendFormat('{{ node }}'),
   ]);
 
-local userNodesIOWait =
+local nfsReqPerOp =
   common.tsOptions
-  + ts.new('iowait % on each node')
+  + ts.new('NFS requests, per operation')
   + ts.standardOptions.withDecimals(0)
   + ts.queryOptions.withTargets([
     prometheus.new(
       '$PROMETHEUS_DS',
       |||
-        sum(rate(node_nfs_requests_total[5m])) by (node)
+        sum(irate(node_nfs_requests_total[5m])) by (method) > 0
       |||
     )
-    + prometheus.withLegendFormat('{{ node }}'),
+    + prometheus.withLegendFormat('{{ method }}'),
   ]);
 
-local userNodesHighNFSOps =
+
+// NFS server diagnostics
+// ----------------------
+local nfsServerCPU =
   common.tsOptions
-  + ts.new('NFS Operation Types on user nodes')
-  + ts.standardOptions.withDecimals(0)
+  + common.tsRequestLimitStylingOverrides
+  + ts.new('NFS server CPU usage')
```

Comment:
<img width="838" height="371" alt="Image" src="https://github.com/user-attachments/assets/0b5ede01-d28f-41d6-a227-c2ffafe6026d" />

Sometimes the limits are way higher than the actual usage, so you can't clearly see the usage metrics clearly. I think this might be okay because you can click on the series you want to isolate, but I wonder if there is a better way to present this information

Reply:
I think we should default to showing CPU requests, because it typically shouldn't be much higher than what you see used -- but if it is, it is an indication of money lost on cloud compute that could be saved. Pods' containers requesting more than they need incur costs, while the same can't be said for very high limits, which are hidden by default.

> wonder if there is a better way to present this information

If it is possible, I think it would be an improvement to be able to switch on/off showing these time series representing cpu/memory-requests/limits and disk-capacity. I'll look into that now!